images/installer/Dockerfile.upi.ci

```diff
@@ -0,0 +1,35 @@
+# This Dockerfile is a used by CI to test UPI platforms for OpenShift Installer
+# It builds an image containing binaries like jq, terraform, awscli, oc, etc. to allow bringing up UPI infrastructure.
+# It also contains the `upi` directory that contains various terraform and cloud formation templates that are used to create infrastructure resources.
+
+
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
+WORKDIR /go/src/github.com/openshift/installer
+COPY . .
+RUN hack/build.sh
+
+FROM centos:7
+COPY --from=registry.svc.ci.openshift.org/ocp/4.0:cli /usr/bin/oc /bin/oc
+COPY --from=builder /go/src/github.com/openshift/installer/bin/openshift-install /bin/openshift-install
+COPY --from=builder /go/src/github.com/openshift/installer/upi /var/lib/openshift-install/upi
+
+## epel-release is required for jq
+## gettext is required for envsubst
+RUN yum install --setopt=tsflags=nodocs -y \
+    epel-release \
+    gettext \
+    openssh-clients && \
+    yum update -y && \
+    yum install --setopt=tsflags=nodocs -y \
+    unzip gzip jq awscli util-linux && \
+    yum clean all && rm -rf /var/cache/yum/*
+
+ENV TERRAFORM_VERSION=0.11.11
+RUN curl -O https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip && unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip -d /bin/
```

Comment:
Hmm, would be nice if we could use the version baked into the installer somehow, although I don't know quite how that would work.  This is fine until we get some ideas ;).

Reply:
I wouldn't bother. these are not shipped to customers. and docs where I use already points to download the `latest`. https://github.com/openshift/installer/pull/1416/commits/3836fcb5c25f2f72eb773c7dfa06f3e8ee65ba04#diff-19bea2d1d6a72e1cf01e6747fa6f66b1R273